lib/smart_answer_flows/simplified-expenses-checker.rb

```diff
@@ -258,6 +261,10 @@ def define
           calculator.vehicle_costs
         end
 
+        precalculate :vehicle_price do
+          calculator.vehicle_price
```

Comment:
This is being used and checked in the integration tests.. An example: `test/integration/smart_answer_flows/simplified_expenses_checker_test.rb:103`